.changeset/beige-moles-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
```

Comment:
```suggestion
'@sumup/circuit-ui': patch
```

Reply:
it was auto-applied by changeset, I have updated it.